.bento/config.yml

```diff
@@ -28,7 +28,7 @@ tools:
     - subprocess-without-shell-equals-true
     - try-except-continue
     - urllib-urlopen
-    run: true
+    run: false
```

Comment:
Can we leave this on and archive the relevant places this fires on this PR

Reply:
done